upup/pkg/fi/task.go

```diff
@@ -25,13 +25,23 @@ import (
 )
 
 type Task interface {
-	Run(*Context) error
+	HasName
```

Comment:
I'd like to get it so node tasks don't have to implement unnecessary boilerplate methods such as `GetName`.